releasenotes/notes/2.0/fix-assign-parameters-ffa284ebde429704.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 fixes:
   - |
-    Fix incorrect behavior in :class:`.CircuitData` in which, upon parameter assignment,
-    we attempted to modify the cached operation inside of a ``PackedInstruction``. Now
-    we instead discard said cache prompting the ``PackedInstruction`` to build a new Python
-    operation should it be needed.
\ No newline at end of file
+    Fixed an incorrect behavior in :class:`.CircuitData` in which, upon parameter assignment,
```

Comment:
We can delete this release note it was released in 1.3.1: https://github.com/Qiskit/qiskit/pull/13559